docs/src/main/docs/config/04_property-mapping.adoc

```diff
@@ -1,6 +1,6 @@
 ///////////////////////////////////////////////////////////////////////////////
 
-    Copyright (c) 2018 Oracle and/or its affiliates. All rights reserved.
+    Copyright (c) 2018,2019 Oracle and/or its affiliates. All rights reserved.
```

Comment:
Very minor nit: Insert a space before "2019" and a comma after it. That's the convention approved by Oracle Legal.

"Copyright (c) 2018, 2019, Oracle and/or its affiliates. All rights reserved."